demo/helper.js

```diff
@@ -0,0 +1,39 @@
+const crypto = require('crypto');
+const fs = require('fs');
+const { readFile } = require('node:fs/promises');
+
+async function loadFile(filepath) {
+  const buffer = await readFile(filepath);
+  return buffer.toString();
+}
+
+async function loadJSON(filepath) {
+  const content = await loadFile(filepath);
+  return JSON.parse(content);
+}
+
+function writeFile(filename, content) {
+  return fs.writeFileSync(filename, content);
+}
+
+function base64encode(input) {
+  return Buffer.from(input).toString('base64url');
+}
+
+function base64decode(input) {
+  return Buffer.from(input, 'base64url').toString();
+}
+
+const hasher = (data) => {
```

Comment:
@ragnika any reason for this to be defined as an anonymous function assigned to the variable and all above by using `function` keyword?

Reply:
@linasi, nah, i just didn't pay enough attention when i was moving stuff around and the scripts into this repo. Thanks for pointing it out